plugin/backend.go

```diff
@@ -79,11 +79,13 @@ func Backend() *GcpAuthBackend {
 			[]*framework.Path{
 				pathConfig(b),
 				pathLogin(b),
+				pathConfigRotateRoot(b),
 			},
 			pathsRole(b),
 		),
-		InitializeFunc: b.initialize,
-		Invalidate:     b.invalidate,
+		InitializeFunc:   b.initialize,
+		Invalidate:       b.invalidate,
+		RotateCredential: b.rotateRootCredential,
```

Comment:
Did we ever land on a decision around what the API will look like when we add support for static roles? Presumably it will be backwards compatible with this one?

Reply:
We haven't landed on a decision, but approaches we have discussed should be backwards compatible. Either a net new addition to the `framework.Backend` to isolate static credential rotation methods (ex `RotateStaticCredentials`). If that feels like overload, we can extend support on this method itself to include static role rotations, and conditionally use either/or based on the rotation job type